.gitignore

```diff
@@ -1,2 +1,3 @@
 build/
 *.user
+src/appMain/sdl_preloaded_pt.json
```

Comment:
@LuxoftAKutsan, I'm recommend to add a comment with information like in commit message